tests/testthat/test-pretty_rel_path.R

```diff
@@ -37,11 +37,6 @@ test_that("Find relative path starting outside of package directory, return abso
   r_version <- as.numeric_version(glue::glue("{R.version$major}.{R.version$minor}"))
   is_windows <- .Platform$OS.type == "windows"
```

Comment:
This should be deleted as well since these variables are only used in `skip_if()` condition.
```suggestion
```

Reply:
Thanks!